service/src/main/kotlin/no/nav/su/se/bakover/service/revurdering/RevurderingServiceImpl.kt

```diff
@@ -916,18 +909,7 @@ internal class RevurderingServiceImpl(
         }
     }
 
-    override fun lagBrevutkast(
-        revurderingId: UUID,
-        fritekst: String,
-    ): Either<KunneIkkeLageBrevutkastForRevurdering, ByteArray> {
-        return hentBrevutkast(revurderingId, fritekst)
-    }
-
-    override fun hentBrevutkast(revurderingId: UUID): Either<KunneIkkeLageBrevutkastForRevurdering, ByteArray> {
-        return hentBrevutkast(revurderingId, null)
-    }
-
-    private fun hentBrevutkast(
+    override fun hentBrevutkast(
```

Comment:
@jacob-meidell Antar at denne tidligere var brukt til både å forhåndsvise et brevutkast og å vise det faktiske brevet. Men nå brukes den kun til å forhåndsvise et brevutkast? Gir det mening å rename denne til hentBrevutkast når det blir generert on the fly basert på en valgfri fritekst?

Mer spesifikt er dette forhåndsvisning av vedtaksbrevet eller?

Reply:
Vi hadde unødvendig 2 funksjoner for brev, som begge kallte den samme funksjonen. 1 av dem tok in fritekst, den andre gjorde ikke.- Kunne like godt bare ha 1 funksjon